Description: Revert switch to'loopdev-3'
 Upstream switched to `loopdev-3` as the `loopdev` crate is now
 unmaintained. However this dependency isn't ready to be included to
 Debian yet, so let's temporarily revert that change.
Author: Arnaud Ferraris <aferraris@debian.org>
Forwarded: not-needed
Last-Update: 2024-02-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -37,9 +37,8 @@
 version = "0.2.139"
 
 [dependencies.loopdev]
-version = "0.5.0"
+version = "0.4.0"
 optional = true
-package = "loopdev-3"
 
 [dependencies.smart-default]
 version = "0.7.1"
